components/booking/DatePicker.tsx

```diff
@@ -9,10 +9,29 @@ import classNames from "@lib/classNames";
 import weekdayNames from "@lib/core/i18n/weekdayNames";
 import getSlots from "@lib/slots";
 
+import { PeriodType } from ".prisma/client";
+
 dayjs.extend(dayjsBusinessDays);
 dayjs.extend(utc);
 dayjs.extend(timezone);
 
+interface DatePickerProps {
```

Comment:
Extract of the period* variables in Prisma, should prove useful for typing:

```javascript
  periodType              String?                @default("unlimited") // unlimited | rolling | range
  periodStartDate         DateTime?
  periodEndDate           DateTime?
  periodDays              Int?
  periodCountCalendarDays Boolean?
  ```
  
  organizerTimeZone: string

Reply:
My bad. This is related to #892 @emrysal